### PR TITLE
docs: July 2024 release notes

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
July 2024 release notes

## How
- **Highlights**: Native authentication & rate limit messaging
- **Platform improvements**: API key management, frequency of sync progress, discover always runs, marketplace announcement
- **Connector enhancements**: MySQL large syncs, S3 speed, Redshift and Snowflake refreshes & RFR, builder additions and UX improvements, platform/connector versioning, pyAirbyte upgrade

## Can this PR be safely reverted and rolled back?
- [ x] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Review by pmossman — APPROVED** [review]

File: docs/release_notes/july_2024.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# July 2024
+
+## airbyte v0.63.5 to v0.63.13
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## ✨ Highlights
+
+[Native authentication](https://github.com/airbytehq/airbyte/issues/41634) by username and password is available in Airbyte OSS so that any instance of Airbyte is secure by default. The newest version of `abctl` (v0.11.0) now generates the credentials randomly and you can use `abctl local credentials` to retrieve them. See our [quickstart docs](using-airbyte/getting-started/oss-quickstart#2-run-airbyte) for more details. For more detailed documentation about authentication, see our docs [here](/deploying-airbyte/integrations/authentication).
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/release_notes/july_2024.md:9]: updated!

File: docs/release_notes/july_2024.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+![Rate Limited Status](./assets/rate_limited.png)
+
+
+## Platform Releases
+
+- Managing and obtaining API keys is now consistent across all our products. We have brought our API portal in-house and API keys can now be generated and found in the “Applications” page of our UI. Along with this, the Terraform provider and Java/Python SDKs have been updated to automatically retrieve tokens between each action, so that users don’t need to worry about fetching tokens themselves.
+
+- The progress of active syncs is now reported every 10 seconds for increased visibility and tracking (available in OSS v.0.63.5)
+
+- Schema discovery is now always run as a part of the sync for Airbyte Cloud. This ensures we always run the sync with the latest schema, which ensures the auto-propagation and backfill features works as expected.
```
Comment (davinchia) [inline docs/release_notes/july_2024.md:22]: thought: is it worth noting we intend to release this to OSS in a few weeks?

File: docs/release_notes/july_2024.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+## ✨ Highlights
+
+[Native authentication](https://github.com/airbytehq/airbyte/issues/41634) by email and password is available in Airbyte OSS so that any instance of Airbyte is secure by default. The newest version of `abctl` (v0.11.0) now generates the credentials randomly and you can use `abctl local credentials` to retrieve them. See our [quickstart docs](using-airbyte/getting-started/oss-quickstart#2-run-airbyte) for more details. For more detailed documentation about authentication, see our docs [here](/deploying-airbyte/integrations/authentication).
+
+We have also released rate limited messaging, providing users more transparency into what's going on with their syncs. When a source is rate limited, it can not only tell the platform but also include a timestamp when the API is expected to be available again to resume extracting records. This is used to tell users the connection is being rate limited, and if the timestamp is provided we include a countdown to when Airbyte will start syncing again.
+
+![Rate Limited Status](./assets/rate_limited.png)
+
+
+## Platform Releases
```
Comment (davinchia) [inline docs/release_notes/july_2024.md:16]: should we also announce the release of Workloads?

**Review by davinchia — APPROVED** [review]
Minor non-blocking comments. :shipit:

**Pull request merged**